src/Math-Tests-Complex/PMComplexTest.class.st

```diff
@@ -162,8 +162,8 @@ PMComplexTest >> testArcTanDenominator [
 	self assert: (c2 arcTan: c1 * c1) equals: Float pi
 ]
 
-{ #category : #tests }
-PMComplexTest >> testArg [
+{ #category : #'testing - polar coordinates' }
+PMComplexTest >> testArgumentOfAPositivePureImaginaryNumber [
```

Comment:
Another test is cos pi/4 + i sin pi/4, whose argument is pi/4. This, and the other test converts fear into boredom and I would write no further tests.

Any objections?